packages/core/src/cloud/utils.ts

```diff
@@ -0,0 +1,18 @@
+import { PlatformApiClient } from "@llamaindex/cloud";
+import { ClientParams, DEFAULT_BASE_URL } from "./types";
+
+export async function getClient({
+  apiKey,
+  baseUrl,
+}: ClientParams = {}): Promise<PlatformApiClient> {
+  // Get the environment variables or use defaults
+  baseUrl = baseUrl ?? process.env.LLAMA_CLOUD_BASE_URL ?? DEFAULT_BASE_URL;
+  apiKey = apiKey ?? process.env.LLAMA_CLOUD_API_KEY;
+
+  const { PlatformApiClient } = await import("@llamaindex/cloud");
```

Comment:
@himself65, we need to do an `await import` here as @llamaindex/cloud is an ESM module.

Unfortunately, we're getting an `ERR_UNSUPPORTED_DIR_IMPORT` error (stack trace below) doing so. 
The `await import` is used a couple of times (e.g. for transformers.js), so it shouldn't be an issue with our setup but with the code generated by Fern.
Any idea? 

```
Error [ERR_UNSUPPORTED_DIR_IMPORT]: Directory import '/Users/marcus/code/llamaindex/LlamaIndexTS/node_modules/.pnpm/@llamaindex+cloud@0.0.1/node_modules/@llamaindex/cloud/lib/api' is not supported resolving ES modules imported from /Users/marcus/code/llamaindex/LlamaIndexTS/node_modules/.pnpm/@llamaindex+cloud@0.0.1/node_modules/@llamaindex/cloud/lib/index.js
    at new NodeError (node:internal/errors:405:5)
    at finalizeResolution (node:internal/modules/esm/resolve:321:11)
    at moduleResolve (node:internal/modules/esm/resolve:980:10)
    at defaultResolve (node:internal/modules/esm/resolve:1193:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:403:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:372:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:249:38)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:76:39)
    at link (node:internal/modules/esm/module_job:75:36) {
  url: 'file:///Users/marcus/code/llamaindex/LlamaIndexTS/node_modules/.pnpm/@llamaindex+cloud@0.0.1/node_modules/@llamaindex/cloud/lib/api',
  code: 'ERR_UNSUPPORTED_DIR_IMPORT'
```